go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.0
 	sigs.k8s.io/controller-tools v0.4.1
 	sigs.k8s.io/kustomize/kustomize/v3 v3.5.4
-	sigs.k8s.io/mcs-api v0.0.0-20200908023942-d26176718973
+	sigs.k8s.io/mcs-api v0.1.0
```

Comment:
Hopefully this is fine :)

Reply:
Good point,

commit d26176718973179fb643f9e7408457ee3430139e (tag: v0.1.0)

I checked, and it seems to be exactly the same point in git :)